### DISCUSSION TITLE
RustDesk on mobile device

### DISCUSSION THREAD

**Discussion by kytril87** [rustdesk/rustdesk#8423] [General]
Hi, I need some advice. I'm using RustDesk on a mobile device. And it happens that I can only connect to it for a certain time. Then the icon changes color to orange and throws an error that the service is not running. While the service is running, the battery has no limitations. In order to connect, I have to unlock my phone and preferably open RustDesk. I also use RustDesk on tablets and there is no such problem. Does anyone have experience with this? Thank you

**Comment by rustdesk** [comment]
https://twitter.com/rustdesk/status/1798387103862231205
